sfn-query-large-datasets-cdk/README.md

```diff
@@ -0,0 +1,110 @@
+# Query large datasets (Amazon Athena, Amazon S3, AWS Glue, Amazon SNS)
+
+**Resources used**  
+
+| Resource Type | Name    |
+|------------------|--------------------|
+| Athena Workgroup | AthenaWorkgroup    |
+| State Machine    | AthenaStateMachine |
+| SNS Topic        | SNSTopic           |
+| S3 Bucket        | DataBucket         |
+| Glue Database    | GlueDatabase       |
+| Glue Crawler     | GlueCrawler        |  
```

Comment:
Great to see an improvement in the markdown with respect to the terraform variant. 

Shall we take this opportunity to also improve the terraform folder? The list of resources used looks much cleaner now.

Reply:
I've asked the person which published the terraform version. Will update it, if I get an approval.